src/Desktop/app.ts

```diff
@@ -0,0 +1,44 @@
+import { resolve } from "path";
+import { app, BrowserWindow } from "electron";
+import { server, url } from "./server";
+
+let main: BrowserWindow | undefined;
+
+const mainWindowUrl = new URL("/hostedExplorer.html", url);
+
+server.listen(url.port);
+
+app.on("ready", createMainWindow);
+app.on("activate", createMainWindow);
+app.on("window-all-closed", onWindowAllClosed);
+
+function createMainWindow() {
+  if (main) {
+    return;
+  }
+
+  // Create the browser window.
+  main = new BrowserWindow({
+    icon: resolve(__dirname, "..", "..", "src", "Desktop", "cosmosdb.ico"),
+    width: 1024,
+    height: 800,
```

Comment:
I know it's WIP, but we probably want to define this as constant so we can make it configurable in the future?